src/python/pants/backend/python/util_rules/pex_from_targets_test.py

```diff
@@ -46,23 +131,14 @@ def test_constraints_validation(rule_runner: RuleRunner) -> None:
             """
             # Comment.
             --find-links=https://duckduckgo.com
-            Foo._-BAR==1.0.0  # Inline comment.
+            # TODO(John Sirois): XXX: File Pex issue and restore `.` in `Foo._-BAR` project name.
```

Comment:
Missed this one. Filed https://github.com/pantsbuild/pex/issues/1302 and will circle back after another Pex fix / upgrade cycle before landing this PR.

Reply:
Now fixed with Pex 2.1.38.